schema-engine/connectors/mongodb-schema-connector/tests/introspection/dirty_data/mod.rs

```diff
@@ -71,16 +70,15 @@ fn mixing_types() {
 
     expected.assert_eq(res.datamodel());
 
-    res.assert_warning_code(101);
-    res.assert_warning(
-        "The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type.",
-    );
+    let expect = expect![[r#"
+        *** WARNING ***
 
-    res.assert_warning_affected(&json!([{
-        "model": "A",
-        "field": "first",
-        "tpe": "Json",
-    }]));
+        The following fields had data stored in multiple types. Either use Json or normalize data to the wanted type:
+
+          - model: A, field: first, type: Json
```

Comment:
I think we usually render these without the empty line in between right now, and probably want to continue doing so.

Reply:
[`b91cf70` (#3906)](https://github.com/prisma/prisma-engines/pull/3906/commits/b91cf7049be8e82aa6916afb356e0a6d6070f96a)